skimage/feature/texture.py

```diff
@@ -214,6 +214,12 @@ def greycoprops(P, prop='contrast'):
     assert num_dist > 0
     assert num_angle > 0
 
+    # normalize each GLMC
```

Comment:
Perhaps add a catch for `normed=False`, that mentions that it now always happens?

Also, add a test that would catch a previously failing case.